src/main/java/uk/gov/hmcts/reform/coh/controller/QuestionRoundController.java

```diff
@@ -135,6 +137,8 @@ public ResponseEntity updateQuestionRound(@PathVariable UUID onlineHearingId, @P
         }
 
         onlineHearing = optionalOnlineHearing.get();
+
+        sessionEventService.createSessionEvent(onlineHearing, EventTypes.QUESTION_ROUND_ISSUED.getEventType());
```

Comment:
Thanks to yet another HLD change, the state of the questions in the round need to be set to "questions_issue_pending" instead of "questions_issued". Only after the notification job actually sends the message to the endpoint does the questions and online hearing get updated

Reply:
I'm so confused